### DISCUSSION TITLE
[WP4861 Discussion]: not all articles from the description are listed in the bibliography

### DISCUSSION THREAD

**Discussion by egonw** [wikipathways/wikipathways-help#125] [pathway-discussion]
### Discussion

I think it would be nice to add all papers from the Description to the bibliography. Let me know if I can assist to make that possible.

### Type of revision

- [ ] Major
- [X] Minor
- [ ] Deletion

**Comment by khanspers** [comment]
Good catch. The remaining two references from the Description has been added to the Bibliography: https://classic.wikipathways.org/index.php/Pathway:WP4861
